sensor/kubernetes/certrefresh/tls_issuer_common.go

```diff
@@ -66,26 +66,41 @@ type tlsIssuerImpl struct {
 	requestOngoing               atomic.Bool
 	newMsgFromSensorFn           newMsgFromSensor
 	requiredCentralCapability    *centralsensor.CentralCapability
+	started                      bool
+	online                       bool
```

Comment:
I think these two should be atomic, `Start`, `Stop`, and `Notify` are called in different routines so there are no guarantees reading and writing these two.

Reply:
Good point, done.